File: Chapter 04/Python/section_4_4_anti_patterns/after_rich_domain_thin_controller/domain/customer.py
class Customer:
    """
    ARCHITECTURE NOTE: Not every domain model needs complex behavior. 
    Because the core business rules for this bounded context revolve 
    around the Order, this Customer class can remain a simple data 
    entity holding state.
    """
    def __init__(self):
        self.id = 0
        self.type = ""  # e.g., "Gold"
        self.email = ""

File: Chapter 04/JS/section_4_4_anti_patterns/after_rich_domain_thin_controller/domain/customer.js
/**
 * ARCHITECTURE NOTE: Not every domain model needs complex behavior. 
 * Because the core business rules for this bounded context revolve 
 * around the Order, this Customer class can remain a simple data 
 * entity holding state.
 */
class Customer {
    constructor() {
        this.id = 0;
        this.type = ""; // e.g., "Gold"
        this.email = "";
    }
}

module.exports = Customer;